scripts/globals/damage/magic_accuracy.lua

```diff
@@ -0,0 +1,179 @@
+-----------------------------------
+xi = xi or {}
+xi.damage = xi.damage or {}
+xi.damage.magicAccuracy = xi.damage.magicAccuracy or {}
+-----------------------------------
+-- Modifier Tables per element.
+local spellAcc          = { xi.mod.FIREACC,               xi.mod.ICEACC,               xi.mod.WINDACC,                xi.mod.EARTHACC,               xi.mod.THUNDERACC,                 xi.mod.WATERACC,              xi.mod.LIGHTACC,           xi.mod.DARKACC           }
+local strongAffinityAcc = { xi.mod.FIRE_AFFINITY_ACC,     xi.mod.ICE_AFFINITY_ACC,     xi.mod.WIND_AFFINITY_ACC,      xi.mod.EARTH_AFFINITY_ACC,     xi.mod.THUNDER_AFFINITY_ACC,       xi.mod.WATER_AFFINITY_ACC,    xi.mod.LIGHT_AFFINITY_ACC, xi.mod.DARK_AFFINITY_ACC }
+local rdmMerit          = { xi.merit.FIRE_MAGIC_ACCURACY, xi.merit.ICE_MAGIC_ACCURACY, xi.merit.WIND_MAGIC_ACCURACY,  xi.merit.EARTH_MAGIC_ACCURACY, xi.merit.LIGHTNING_MAGIC_ACCURACY, xi.merit.WATER_MAGIC_ACCURACY }
+
+function xi.damage.magicAccuracy.calculateCasterMagicAccuracy(caster, target, spell, skillType, spellElement, statDiff)
```

Comment:
Can all of these new functions be in the `xi.name.name = function(arg, arg)` format please?